library/src/main/java/pl/rspective/voucherify/android/client/api/VoucherifyApi.java

```diff
@@ -1,12 +1,13 @@
 package pl.rspective.voucherify.android.client.api;
 
+import pl.rspective.voucherify.android.client.exception.VoucherifyError;
 import pl.rspective.voucherify.android.client.model.VoucherResponse;
 import retrofit.http.GET;
 import retrofit.http.Query;
 
 public interface VoucherifyApi {
 
     @GET("/client/v1/validate")
-    VoucherResponse validateVoucher(@Query("code") String code, @Query("tracking_id") String trackingId, @Query("channel") String channel);
+    VoucherResponse validateVoucher(@Query("code") String code, @Query("tracking_id") String trackingId, @Query("channel") String channel) throws VoucherifyError;
```

Comment:
I'm not sure if putting here `throws VoucherifyError` is proper. Do we need this as checked exception?